modules/button/react/lib/IconButton.tsx

```diff
@@ -258,5 +274,30 @@ const getIconButtonColors = (
           icon: toggled ? themePrimary.main : 'rgba(255, 255, 255, 0.75)',
         },
       };
+
+    case IconButtonVariant.ToolbarSquare:
+      return {
+        default: {
+          icon: toggled ? themePrimary.dark : colors.licorice200,
```

Comment:
Why not `themePrimary.main`? All of the other icons use blueberry400, not blueberry500.

Reply:
Design updated this to 500 because when active, it also have a blue background, this allows the icon to stand out a little bit more